rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -2511,4 +2511,21 @@ class ReduceSpec extends FlatSpec with Matchers with AppendedClues with Persiste
 
     task.runSyncUnsafe(timeout)
   }
+
+  "term split size" should "be limited to Short.MaxValue" in {
+    implicit val errorLog = new ErrorLog[Task]()
+    val sends             = Seq.fill(Short.MaxValue + 2)(Send())
+    val proc              = Par(sends)
+
+    val result = withTestSpace(errorLog) {
+      case TestFixture(_, reducer) =>
+        implicit val env = Env[Par]()
+        val task         = reducer.eval(proc)
+        Await.result(task.failed.runToFuture, 0.seconds)
```

Comment:
The timeout here gives me chills. If it's really that snappy, let's at least have 1.second

Reply:
This is on purpose. It's really like sync operation, if terms size is to large, it should fail immediately.

But for sure it depends on internal implementation if 0 sec. can sometimes fail. Can `10.milis` be more than enough for a method call?